contracts/Core/BlockManager.sol

```diff
@@ -197,15 +172,9 @@ contract BlockManager is Initializable, ACL, BlockStorage {
             lastVisited = sorted[i];
             accWeight = accWeight + (voteManager.getVoteWeight(epoch, assetId, sorted[i]));
```

Comment:
Hey
When Staker proposes block, the medians they propose should be weighted on basis of influence, right?
If yes 
We have to change vote properties, like VoteWeights, right now it stores stake, we need influence there

Reply:
Correct. Fixed in https://github.com/razor-network/contracts/pull/218/commits/f8ccf9ce00ecce17230bed1d2a9199464358e940